src/radical/pilot/task_description.py

```diff
@@ -708,6 +708,10 @@ def _verify(self):
         if self.gpu_threads     : pass
         if self.gpu_thread_type : pass
 
+        if self.environment:
+            for k,v in self.environment.items():
+                self.environment[k] = str(v)
```

Comment:
we shouldn't redefine this (kinda a duplication), since the corresponding schema already deals with it `ENVIRONMENT     : {str: str}`, thus when `td.verify()` is called it will convert it.

Reply:
Ah, you are right - I was on the wrong track here, thanks.